### PR TITLE
Add some pretty log commands

### PR DESCRIPTION
**Pull request by marcusphi**

### CODE REVIEW

**Comment by marcusphi** [comment]
I made this repo public to not waste our private quota on not so private stuff. I hope you do not mind.

File: README.md [hunk 1]
```diff
@@ -199,6 +199,18 @@ $ git status
 
     ```$ git show HEAD~1```
 
+4. A really nice colorized tree view: 
+
+    ```$ git log --graph --pretty=format:'%C(yellow)%h%Creset -%C(red)%d%Creset %s %Cgreen(%cd) %C(bold blue)<%an>%Creset' --abbrev-commit --date=local```
+
+5. This is a very long command so let's add this as an alias 'l', by doing:
+
+    ```$ git config --global --add alias.l "log --graph --pretty=format:'%C(yellow)%h%Creset -%C(red)%d%Creset %s %Cgreen(%cd) %C(bold blue)<%an>%Creset' --abbrev-commit --date=local"```
```
Comment (RobertDahlstrom) [inline README.md]: Perhaps the alias should be named something else than just 'l'. E.g. tree-log?

As this is intended to be a tutorial and as easy to understand as possible

**Comment by RobertDahlstrom** [comment]
> I made this repo public to not waste our private quota on not so private stuff. I hope you do not mind.

Not at all, the intention was always to make this public, just wanted it to be a bit better first.

**Review by RobertDahlstrom — COMMENTED** [review]
Please give feedback on comment about alias name

**Review by RobertDahlstrom — APPROVED** [review]

**Pull request merged**